tests/huawei_vrp/display_lldp_neighbor/huawei_vrp_display_lldp_neighbor4.yml

```diff
@@ -0,0 +1,105 @@
+---
+parsed_sample:
+  - aggregation_port_id: ""
+    asset_tracking_identifier: ""
+    autoclass_completed: "Autoclass idle"
+    autoclass_request: "Autoclass idle"
+    autonegotiation_enabled: "Yes"
+    autonegotiation_supported: "No"
+    capabilities: "bridge"
+    chassis_id: "7c0e-ce50-8380"
+    chassis_type: "MAC Address"
+    discovered_time: "2024-10-10 01:13:32"
+    dual_signature_power_classx_mode_a: "Reserved"
+    dual_signature_power_classx_mode_b: "Reserved"
+    echo_receive_tw: "0"
+    echo_transmit_tw: "0"
+    energy_efficient_ethernet_support: "No"
+    expired_time: "99"
+    fallback_receive_tw: "0"
+    firmware_revision: ""
+    hardware_revision: ""
+    link_aggregation_enabled: "--"
+    link_aggregation_supported: "--"
+    local_interface: "GE1/0/25"
+    location_format: ""
+    location_information: ""
+    manufacturer: ""
+    manufacturer_name: ""
+    maximum_frame_size: ""
+    med_capability: ""
+    med_device_class: ""
+    med_device_type: ""
+    media_policy_dscp: ""
+    media_policy_l2_priority: ""
+    media_policy_type: ""
+    media_policy_vlan_id: ""
+    mgmt_address: "10.21.100.100"
+    mgmt_address_type: ""
+    model_name: ""
+    neighbor_description:
+      - "Huawei AP AP111"
+      - "Huawei Versatile Routing Platform Software"
+      - "VRP (R) software, Version 1.111 (AP111 V000R000C00SPC000)"
+      - "Copyright (C) 2011-2024 Huawei Technologies Co.,Ltd"
+      - "Cisco IOS Software, C3750E Software (C3750E-UNIVERSALK9-M), Version 12.2(58)SE2,
+        RELEASE SOFTWARE (fc1)"
+      - "Technical Support: http://www.cisco.com/techsupport"
+      - "Copyright (c) 1986-2011 by Cisco Systems, Inc."
+      - "Compiled Thu 21-Jul-11 01:23 by prod_rel_team"
```

Comment:
@evilmonkey19
This might be a "logic error bug" since the Huawei VRP is on GE1/0/11 and the Cisco 3750 is on GE1/0/25.

The output for VRP and 3750 aren't supposed to be together, right?

Thank you for investigating and following up on this.